core/src/test/java/org/infinispan/api/AsyncAPITest.java

```diff
@@ -14,21 +16,29 @@
 import java.util.concurrent.TimeUnit;
 
 import static org.junit.Assert.*;
+import static org.testng.AssertJUnit.assertTrue;
 
 @Test(groups = "functional", testName = "api.AsyncAPITest")
 public class AsyncAPITest extends SingleCacheManagerTest {
 
    private Cache<String, String> c;
-
+   private ControlledTimeService timeService = new ControlledTimeService();
    private Long startTime;
 
    @Override
    protected EmbeddedCacheManager createCacheManager() throws Exception {
       EmbeddedCacheManager cm = TestCacheManagerFactory.createCacheManager(false);
+      TestingUtil.replaceComponent(cm, TimeService.class, timeService, true);
       c = cm.getCache();
       return cm;
    }
 
+   @Override
+   protected void teardown() {
+      TestingUtil.replaceComponent(cacheManager, TimeService.class, new DefaultTimeService(), true);
```

Comment:
Please do, I'm not sure why `CacheNotifierTest` does it but it shouldn't be necessary.